src/utils/nodeTemplates.ts

```diff
@@ -269,7 +276,7 @@ export function createDefaultConstructor(node: ContractDefinition, ast: AST): Fu
 
 export function createVariableDeclarationStatement(
   varDecls: (VariableDeclaration | null)[],
-  intitalValue: Expression,
+  intitalValue: Expression | undefined,
```

Comment:
I don't think this is needed

Reply:
That is the type of `initialValue` for a `VariableDeclarationStatement`